### ISSUE TITLE
Not able to block MS-Edge and not all app appear in simplewall

### ISSUE THREAD

**Issue by Mannshoch** [henrypp/simplewall#917]
I tested Simple wall with three apps.
Firefox, MS edge and Installing Adobe Reader. Edge and Adobe Reader do not appear to be requested to be allowed or blocked.  
What ever I do, MS-Edge seems not blocked. 
If I download Adobe Reader web installer and run it after that, there is no question about if the web installer is allowed to access Internet. Also it seems that after installing Adobe Reader also Adobe reader have access to Internet.

**Comment by henrypp** [comment]
No telepaths.

**Comment by Mannshoch** [comment]
I uninstalled simple wall completely and reinstalled it again. Then I set up all again. I may detect the problem. May I misunderstood this inconsiderable little "Disable Windows Firewall" tick.
![grafik](https://user-images.githubusercontent.com/7795891/117473061-daa30900-af59-11eb-92aa-949b7a86e446.png)

From the Description:
> Simple tool to configure Windows Filtering Platform (WFP) which can configure network activity on your computer.

I assumed that this is a kind of add-on with some new features for the Windows Firewall.

But If I understand it right now, It replaces the whole Windows Firewall but use its API?
